_blog.yml

```
@@ -5215,3 +5215,14 @@
     - nlp
     - community
     - model
+
+- local: modernbert
+  title: "Finally, a Replacement for BERT: Introducing ModernBERT"
+  author: bclavie
+  guest: true
+  thumbnail: /blog/assets/modernbert/thumbnail.png
+  date: December 19, 2024
+  tags:
+    - nlp
+    - community
+    - research
```

```suggestion
    - research
    - open-source-collab
```
I think this one is fitting as well.